libs/langgraph/tests/__snapshots__/test_pregel_async.ambr

```diff
@@ -1,226 +1,4 @@
 # serializer version: 1
-# name: test_in_one_fan_out_state_graph_waiting_edge_custom_state_class
```

Comment:
why are we removing tests?

Reply:
this was before i realized that we wanted to support both, i have a todo above to bring this back